conda_smithy/linter/lints.py

```diff
@@ -359,38 +359,46 @@ def lint_recipe_v1_noarch_and_runtime_dependencies(
 def lint_noarch_and_runtime_dependencies(
     noarch_value, meta_fname, forge_yaml, conda_build_config_keys, lints
 ):
-    if noarch_value is not None and os.path.exists(meta_fname):
```

Comment:
I inverted the logic here so we can just have an early return, and that saves the rest of the function from being indented.